cmds/provisiond/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"github.com/threefoldtech/zbus"
+	"github.com/threefoldtech/zosv2/modules/provision"
+)
+
+const module = "container"
```

Comment:
```suggestion
const module = "provision"
```

Reply:
I removed the const, it's not used since provisiond is not a zbus service. it only uses the client to do it's work